src/libs/cropOrRotateImage/utils.ts

```diff
@@ -0,0 +1,16 @@
+import {SaveFormat} from 'expo-image-manipulator';
+
+function getSaveFormat(type: string) {
+    switch (type) {
+        case 'image/png':
+            return SaveFormat.PNG;
+        case 'image/webp':
+            return SaveFormat.WEBP;
+        case 'image/jpeg':
+            return SaveFormat.JPEG;
+        default:
+            return SaveFormat.JPEG;
+    }
+}
+
+export default getSaveFormat;
```

Comment:
If we're using `export default` can we please rename the file from `utils.ts` to the method name?